enginetest/join_planning_tests.go

```diff
@@ -1243,6 +1243,159 @@ join uv d on d.u = c.x`,
 			},
 		},
 	},
+	{
+		name: "range join vs good lookup join regression test",
+		setup: []string{
+			"create table vals (val int, filter1 int, filter2 int, filter3 int)",
+			"create table ranges (min int, max int, filter1 int, filter2 int, filter3 int, key filters (filter1, filter2, filter3))",
+			"insert into vals values (0, 0, 0, 0), " +
```

Comment:
we can inject cardinalities into costing, maybe a bit easier than manually writing 100 insertions.